packages/manager/apps/vrack/webpack.config.js

```diff
@@ -6,8 +6,11 @@ module.exports = (env = {}) => {
   const { config } = webpackConfig({
     template: './src/index.html',
     basePath: './src',
+    lessPath: [
+      './node_modules',
+    ],
     root: path.resolve(__dirname, './src'),
-  }, process.env.REGION ? Object.assign(env, { region: process.env.REGION }) : env);
+  }, env);
```

Comment:
Please add:
```js
const REGION = `${_.upperCase(env.region || process.env.REGION || 'EU')}`;
```

and then send the region:
```suggestion
  }, REGION ? Object.assign(env, { region: REGION }) : env);
```

Reply:
This is done. Thanks :)